server/routes/appointments/appointmentsController.test.ts

```diff
@@ -1351,39 +1351,78 @@ describe('Adding post delivery session feedback', () => {
   })
 
   describe('GET /service-provider/action-plan/:actionPlanId/appointment/:sessionNumber/post-session-feedback/edit/:draftBookingId/attendance', () => {
```

Comment:
github compare makes this look a mess, but essentially this moves the existing test under a `When the draft booking is found` block, and adds `When the draft booking is not found` test

```
describe('GET /service-provider/action-plan/:actionPlanId/appointment/:sessionNumber/post-session-feedback/edit/:draftBookingId/attendance', () => {
    describe('When the draft booking is found', () => {
      it('renders a page with which the Service Provider can record the Service user‘s attendance', async () => {
      })
    })

    describe('When the draft booking is not found', () => {
      it('renders a no longer available page', async () => {
      })
    })
  })
```